### PR TITLE
zh-translation: /docs/tasks/observability/kiali/index.md

### PR DESCRIPTION
**Pull request by arunfung**
ref: https://github.com/servicemesher/istio-official-translation/issues/1308

### CODE REVIEW

File: content/zh/docs/tasks/observability/kiali/index.md [hunk 1]
```diff
@@ -98,126 +91,107 @@ $ kubectl apply -f $HOME/istio.yaml
...
+为了获得使用 Kiali 的最佳体验，请在应用程序组件上使用元数据标签 `app` 和 `version` 。 作为模板，Bookinfo 示例应用程序遵循此约定。
 
-## Cleanup
+## 清理{#cleanup}
 
-If you are not planning any follow-up tasks, remove the Bookinfo sample application and Kiali from your cluster.
+如果您不计划任何后续任务，请从群集中删除 Bookinfo 示例应用程序和 Kiali。
 
-1. To remove the Bookinfo application, refer to the [Bookinfo cleanup](/docs/examples/bookinfo/#cleanup) instructions.
+1. 要删除 Bookinfo 应用程序，请参阅 [Bookinfo cleanup](/docs/examples/bookinfo/#cleanup) 说明。
```
Comment (rootsongjc) [inline content/zh/docs/tasks/observability/kiali/index.md]: 链接前请加 `/zh`。

Comment (arunfung) [reply content/zh/docs/tasks/observability/kiali/index.md]: 已修改

**Comment by tanjunchen** [comment]
please fix conclicts.Thanks

**Pull request merged**